### PR TITLE
handle selected dimensions in metadata - update to use new function

### PR DESCRIPTION
**Pull request by andre-urbani**
### What

Amend to handle selected dimensions in metadata
Trello - https://trello.com/c/FEfoLoo6/5675-amend-dp-cantabular-xlsx-exporter-to-handle-selected-dimensions-in-metadata

### How to review

Run tests and confirm code changes are correct

### Who can review

Anyone

### CODE REVIEW

**Review by PauloPortugal — APPROVED** [review]
LGTM

File: schema/schema.go [hunk 1]
```diff
@@ -12,7 +12,16 @@ var cantabularCsvCreated = `{
     {"name": "dataset_id", "type": "string", "default": ""},
     {"name": "edition", "type": "string", "default": ""},
     {"name": "version", "type": "string", "default": ""},
-    {"name": "row_count", "type": "int", "default": 0}
+    {"name": "row_count", "type": "int", "default": 0},
+    {
+      "name": "dimensions_id",
```
Comment (ChrysmOre) [inline schema/schema.go:17]: dimensions_id, should this be dimension_ids?

**Review by saminahbab — DISMISSED** [review]
LGTM

File: schema/schema.go (hunk 1 above)
Comment (j-s-rawat) [reply schema/schema.go:17]: I just added 'id' to make it very obvious that its 'id' and not name or label or any thing else. what ever you guys feel like I am ok with that

**Review by PauloPortugal — APPROVED** [review]

File: schema/schema.go (hunk 1 above)
Comment (PauloPortugal) [reply schema/schema.go]: Who is changing the `csv-exporter` and `matadata-exporter`?

**Pull request merged**